src/pages/docs/packages/frint-props-react.md

```diff
@@ -0,0 +1,5 @@
+---
+title: frint-props-react
+importContentFromUrl: "https://raw.githubusercontent.com/frintjs/frint-props/master/packages/frint-props-react/README.md"
```

Comment:
I am thinking if we can have a generic key with paths like this, for all the markdown imports:

```yaml
importFromGitHub: "frintjs/frint-props/master/packages/frint-props-react/README.md"
```

Reply:
Yeah, That sounds good, I'll apply it.